### PR TITLE
Document how to run Python examples

### PR DESCRIPTION
**Pull request by emilk**
There was documentation there, but I couldn't understand it. Thankfully Jeremy told me the magic incantation, so I decided to write it down in a way that I can understand.

I decided to remove the `-e examples` part of it, since everything seems to work without it and because this is the docs for the `examples` environment says it is WIP:

https://github.com/rerun-io/rerun/blob/0ac38a4df54c45f2c34f6a75c6044f9785719615/pixi.toml#L39-L40

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/{{pr.number}}?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/{{pr.number}}?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/{{pr.number}})
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: examples/python/README.md [hunk 1]
```diff
@@ -3,14 +3,42 @@ The simplest example is [`minimal`](minimal/minimal.py). You may want to start t
...
+## Running our examples with `pixi`
+The easiest way of running our examples from within the Rerun repository is to use [pixi](https://github.com/prefix-dev/pixi?tab=readme-ov-file#installation).
+
+### Building from source
+
+```sh
+pixi shell
+pixi run py-build
+pip install examples/python/minimal
+minimal
```
Comment (abey79) [inline examples/python/README.md:16]: well that's not really the intended way. All examples are already installed in the `examples` environment, which just needs to have a compatible rerun-sdk there, so:

```
pixi run -e examples py-build
pixi run -e examples minimal
```

File: examples/python/README.md [hunk 2]
```diff
@@ -3,14 +3,42 @@ The simplest example is [`minimal`](minimal/minimal.py). You may want to start t
...
+```
+
+### Using pre-build `rerun-sdk`
+For the released version of the SDK, use:
+```shell
+pixi shell
+pip install rerun-sdk==0.15.1
+pip install examples/python/minimal
+minimal
```
Comment (abey79) [inline examples/python/README.md:25]: Likewise.

```
pixi run -e examples pip install rerun-sdk==0.15.1
pixi run -e examples minimal
```

**Review by abey79 — CHANGES_REQUESTED** [review]
Although that might work, I'm not convinced it's simpler, and it's certainly not how it was intended.

**Comment by emilk** [comment]
It seems like @abey79 convinced @jleibs that `pixi shell` is not the best recommendation, and that the summary should be

```
pixi run -e examples py-build
pixi run -e examples minimal
```

**Comment by abey79** [comment]
> It seems like @abey79 convinced @jleibs that `pixi shell` is not the best recommendation, and that the summary should be
> 
> ```
> pixi run -e examples py-build
> pixi run -e examples minimal
> ```

To clarify, `pixi shell` is fine. What I find inappropriate is to advertise using pixi's `default` environment as a recipient to install random stuff, including examples. Our `pixi.toml` declares an `examples` environment that is made for that, and offers isolation from the other pixi environments.

So both are perfectly fine imo:
```
pixi run -e examples py-build
pixi run -e examples minimal
```
and:
```
pixi shell -e examples
pixi run py-build
pixi run minimal
```

The latter case works because `pixi run` recognises that one of its environment has been activated and uses it instead of `default`.

**Comment by emilk** [comment]
Closed by https://github.com/rerun-io/rerun/pull/6466#pullrequestreview-2089971006

**Pull request closed without merging**